.github/workflows/build-and-test.yml

```
@@ -94,6 +94,11 @@ jobs:
           git clone https://github.com/precice/precice.git precice-core
           mkdir -p precice
           cp precice-core/src/precice/SolverInterface.hpp precice/SolverInterface.hpp
+          cd precice-core
+          mkdir build && cd build
+          cmake .. -DPRECICE_MPICommunication=OFF -DPRECICE_PETScMapping=OFF -DPRECICE_PythonActions=OFF -DBUILD_TESTING=OFF
+        env:
+          PKG_CONFIG_PATH: "precice-core/build/lib/pkgconfig"
```

The one you need is in the build directory. The other one is for the installation.
```suggestion
          PKG_CONFIG_PATH: "precice-core/build"
```